src/ArborX_DistributedSearchTree.hpp

```diff
@@ -121,14 +121,28 @@ DistributedSearchTree<DeviceType>::DistributedSearchTree(
   Kokkos::View<Box *, DeviceType> boxes(
       Kokkos::ViewAllocateWithoutInitializing("rank_bounding_boxes"),
       comm_size);
-  // FIXME when we move to MPI with CUDA-aware support, we will not need to
-  // copy from the device to the host
+#ifdef ARBORX_USE_CUDA_AWARE_MPI
+  if (!_bottom_tree.empty())
+  {
+    Kokkos::View<Box const, DeviceType, Kokkos::MemoryTraits<Kokkos::Unmanaged>>
+        root_bounding_volume(&this->getBoundingVolume(this->getRoot()));
```

Comment:
Yes, fascinating that `_bottom_tree.getBoundingVolume` doesnt work here. At least the compiler is very clear about it. :slightly_smiling_face: